paramak/parametric_reactors/eu_demo_2015_reactor.py

```diff
@@ -0,0 +1,421 @@
+
+import paramak
+import numpy as np
+
+
+class EuDemoFrom2015PaperDiagram(paramak.Reactor):
+    """Creates geometry of a simplified EU DEMO model based on the published
+    diagram in Figure 2 of Definition of the basic DEMO tokamak geometry based
+    on systems code studies. Published in Fusion Engineering and Design
+    http://dx.doi.org/10.1016/j.fusengdes.2015.06.097 . Coordinates extracted
+    from the figure are not exact and therefore this model does not perfectly
+    represent the reactor.
+
+    Arguments:
+        rotation_angle (float): the angle of the sector that is desired.
+            Defaults to 360.0.
```

Comment:
```suggestion
        number_of_tf_coils: the number of tf coils to include in the model
```

Reply:
Ah yes forgot that one. Don't we want to keep the Defaults to 360.0 though ?

Reply:
The default is in the argument and appears with type hinting. I think we only need to add "Defaults to X" when there are specific consequences. Otherwise we are just duplicating text. There are a few cases where we have Defaults to None which require further explanation but in this case I think in this case the default in the args is fine.

Happy to go with whatever you like but that is my train of thought